### DISCUSSION TITLE
How to order pcb and parts

### DISCUSSION THREAD

**Discussion by krishy2** [dzurikmiroslav/esp32-evse#38] [Q&A] [answered]
I've visited the EasyEDA site, but I was unable to check the parts and the pcb. Would you confirm that it is working?

**Comment by dzurikmiroslav** [comment, marked as answer]
Hi, from EasyEDA I was order only pcb... Parts I was bought at local shop and aliexpress, and soldering it at home :)
I tested my wallbox based on esp32s2-evse-d-a with Skoda Enyaq and VW e-Up (can you see it in video https://www.youtube.com/shorts/r6YkWEet1aA).

**Reply by krishy2** [reply]
That is an answer why the parts not working :)

**Comment by andrei-tatar** [comment]
I would be willing to pay for an assembled board. Is that something that you're considering?

**Reply by dzurikmiroslav** [reply]
I would like to design a device with ESP32-S3 with features like **ESP32-S2 EVSE DIY ALPHA** but with SMD components and offer it on Tindie... But unfortunately I don't have time for that right now :-( I hope that one day I will have space for that.
